mosdef_cassandra/writers/inp_functions.py

```diff
@@ -241,7 +241,7 @@ def generate_input(system, moves, run_type, run_length, temperature, **kwargs):
             box_dims = np.hstack((box.lengths, box.angles))
         box_matrix = convert_box.convert_to_boxmatrix(box_dims)
         boxes.append(box_matrix)
-    inp_data += get_box_info(boxes)
+    inp_data += get_box_info(boxes, moves)
```

Comment:
Do you have an opinion about passing the full moves object vs. just the restricted insertion details? I'm just thinking from a readability of the code standpoint it might be clearer to see exactly what part of the moves is need in writing the box info?